KonomiAI/Chronicle
# feat: analytics UI

### Related Issue(s)



resolves #263 



### Quick overview of the change

![image](https://user-images.githubusercontent.com/11084692/195198635-6f79d722-79d3-4241-8b1d-04e7c2dcaeae.png)


- Added UI for analytics feature

### Quick testing guide

- Login to the app
- Click on `Analytics` on the side bar
- Make sure you have some data in the system (activity entries)
